src/components/Avatar/Avatar.tsx

```diff
@@ -9,15 +9,15 @@ import {Image} from '../Image';
 import styles from './Avatar.scss';
 import * as avatars from './images';
 
-export type Size = 'small' | 'medium' | 'large';
+type Size = 'small' | 'medium' | 'large';
```

Comment:
This sort of thing crops up a lot. Using this particular case as an example, `Size` was exported here but was never reexported by `src/Avatar/index.ts` and was never used in any other file, so there is no value in exporting it at all.